src/App.tsx

```diff
@@ -1,7 +1,39 @@
-import React from 'react';
+import { Component } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: string | boolean,
```

Comment:
Is it really need to be string or boolean? Can't it be only string type?
You can check if it's an empty string, which means nothing was pressed